integration-tests/pom.xml

```diff
@@ -381,7 +382,7 @@
                   **/ItPodTemplates,
                   **/ItRemoteConsole,
                   **/ItServerStartPolicy,
-		  **/ItServiceStartPolicyDynamicCluster,
+		          **/ItServiceStartPolicyDynamicCluster,
```

Comment:
No extra space needed here

Reply:
this test suite is existing one, not mine from this change. I made change when I saw it's indent incorrectly, I used tab to indent that works always to me. Let me try to fix it again